src/thorin/util/utility.h

```diff
@@ -98,7 +98,7 @@ class TaggedPtr {
 private:
 #if defined(__x86_64__) || (_M_X64)
     int64_t ptr_   : 48; // sign extend to make pointer canonical
-    I       index_ : 16;
+    int64_t index_ : 16;
```

Comment:
OK then, let's stick to ```int64_t```. ```TaggedPtr``` is currently not used within Thorin but this will change.